src/cloudstorage/drivers/local.py

```diff
@@ -609,3 +649,48 @@ def validate_signature(self, signature):
     _PUT_OBJECT_KEYS = {
         'metadata': 'meta_data',
     }
+
+
+class xattr_Windows:
```

Comment:
Can you rename this to `XattrWindows`

https://www.python.org/dev/peps/pep-0008/#class-names